packages/xo-web/src/common/render-xo-item.js

```diff
@@ -226,18 +226,33 @@ export const Sr = decorate([
       container: getContainer(state, props),
     })
   }),
-  ({ id, sr, container, link, newTab, spaceLeft, self }) => {
+  ({
+    container,
+    id,
+    link,
+    newTab,
+    self,
+    showAllocationStrategy,
+    spaceLeft,
+    sr,
+  }) => {
     if (sr === undefined) {
       return unknowItem(id, 'SR')
     }
 
     return (
       <LinkWrapper link={link} newTab={newTab} to={`/srs/${sr.id}`}>
         <Icon icon='sr' /> {sr.name_label}
-        {!self && spaceLeft && isSrWritable(sr) && (
-          <span className={!link && 'text-muted'}>{` (${formatSize(
-            sr.size - sr.physical_usage
-          )} free)`}</span>
+        {!self && (showAllocationStrategy || spaceLeft) && isSrWritable(sr) && (
+          <span className={!link && 'text-muted'}>
+            {' ('}
+            {spaceLeft && `${formatSize(sr.size - sr.physical_usage)} free`}
+            {spaceLeft && showAllocationStrategy && ' - '}
+            {showAllocationStrategy &&
+              sr.allocationStrategy !== undefined &&
+              sr.allocationStrategy}
+            )
+          </span>
```

Comment:
What do you think about grouping these in a single optional group instead of 2?

Reply:
thin/thick is only important in backups. Should it always be displayed with the `spaceLeft`? 
I think, no. So, we keep it in 2 groups.